debezium-core/src/main/java/io/debezium/relational/ColumnEditor.java

```diff
@@ -39,6 +39,14 @@
     int jdbcType();
 
     /**
+     * Get the underlying type for this column where
+     * jdbcType() itself is the generic Types.ARRAY.
+     *
```

Comment:
given the return type is a privitive int we shouldn't really be returning a null.  I will document that it returns a default value when unset.